Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -1,11 +1,11 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras/2.0.31">
+﻿<Project Sdk="MSBuild.Sdk.Extras/2.0.41">
   <PropertyGroup>
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">netstandard1.0;netstandard2.0;Xamarin.iOS10;Xamarin.TVOS10;Xamarin.WatchOS10;MonoAndroid60;MonoAndroid70;MonoAndroid71;MonoAndroid80;MonoAndroid81;MonoAndroid90;uap10.0.16299;tizen40;</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.0;netstandard2.0;Xamarin.iOS10;Xamarin.TVOS10;Xamarin.WatchOS10;MonoAndroid60;MonoAndroid70;MonoAndroid71;MonoAndroid80;MonoAndroid81;MonoAndroid90;tizen40;</TargetFrameworks>
     <AssemblyName>Xamarin.Essentials</AssemblyName>
     <RootNamespace>Xamarin.Essentials</RootNamespace>
-    <PackageId>Xamarin.Essentials</PackageId>
-    <PackageIconUrl>https://raw.githubusercontent.com/xamarin/Essentials/master/Assets/xamarin.essentials_128x128.png</PackageIconUrl>
+    <PackageId>TESTXamarin.Essentials</PackageId>
```

Comment:
let's change this to "DotNetEssentials" (I have just set the secret NUGET_API_KEY in the repo to my API key that is able to upload packages to https://www.nuget.org/packages/DotNetEssentials , so let's replace that too in the call to nuget.bat )

Reply:
Done.

Reply:
you didn't change this to DotNetEssentials, you left it as Xamarin.Essentials

Reply:
Sorry, I left the original name insted of the new one you suggested.